rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2579,6 +2592,13 @@ void game_list_frame::PopulateGameList()
 			}
 		}
 
+		QString directory_size = tr("Unknown");
+
+		if (usz size = game->info.size_on_disk; size != umax)
+		{
+			directory_size = gui::utils::format_byte_size(size);
+		}
```

Comment:
this can be a ternary in the constructor below. also, the local here isn't really necessary either

Reply:
Ended u with a local anyway because 3 references to game->info.size_on_disk is really long.